### DISCUSSION TITLE
登录过期时间好像很短啊

### DISCUSSION THREAD

**Discussion by xiaodiao216** [hslr-s/sun-panel#286] [Q&A]
登录过期时间怎么修改啊？希望不要让我们去改代码，能在后台设置里面直接设置最好。

**Comment by sunjx97** [comment]
https://doc.sun-panel.top/zh_cn/faq/faq.html#%E7%99%BB%E5%BD%95%E6%97%B6%E6%95%88%E6%98%AF%E5%A4%9A%E4%B9%85-%E6%80%BB%E6%98%AF%E8%AE%A9%E6%88%91%E9%A2%91%E7%B9%81%E7%99%BB%E5%BD%95
